### DISCUSSION TITLE
Camel case column name in sql query being automatically set to lowercase

### DISCUSSION THREAD

**Discussion by swanjeezy** [apache/datafusion#6187] [Q&A] [answered]
I've noticed that the column names in sql queries get automatically set in lower case. For example, testId becomes testid. This is causing some issues with the sql query executing on the data. 

Ex error msg: SchemaError(FieldNotFound { field: Column { relation: None, name: "testid"}, valid_fields" [Column { relation: Some(Bare{ table: "tableName" }), name: "testId"}] })

Is there some sort of option that can turn this off or is this a bug? I noticed that there's a config option datafusion.sql_parser.enable_ident_normalization, but I'm unsure what this does. Any help would be greatly appreciated!

**Comment by bubbajoe** [comment, marked as answer]
`datafusion.sql_parser.enable_ident_normalization` set to false will fix this issue

**Reply by alamb** [reply]
All configuration settings can be found at https://arrow.apache.org/datafusion/user-guide/configs.html
